runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkRuntimeContext.java

```diff
@@ -114,35 +118,6 @@ public CoderRegistry getCoderRegistry() {
     return coderRegistry;
   }
 
-  private Coder<?> getCoder(Combine.CombineFn<?, ?, ?> combiner) {
```

Comment:
LGTM - this change seems to keep the same functionality with far less lines of code so 👍 👍.